### ISSUE TITLE
Need conda install

### ISSUE THREAD

**Issue by Huangtianyu-caas** [RetroWWU/paPAML#2]
Dear developer,

Thank you for your wonderful software which allows a faster running of PAML software.

Would it be possible to release paPAML to the conda install? Because its hard for a "normal" user to install it in a Linux environment.

For example, when I run "perl paPAML.pl", an issue came out: "Can't locate Proc/ProcessTable.pm in @INC (you may need to install the Proc::ProcessTable module)". It is impossible for me to install the Proc::ProcessTable module, as I do not have the root authority.

Please kindly consider it.
Kind regards,

**Comment by roderick-mao** [comment]
yes, if we use on the cluster. we are not possibly to use as root.

**Comment by RetroWWU** [comment]
Hello, I added a description about using Conda - I checked it on FreeBSD and Linux / Ubuntu - but sorry it did not seem to work on OsX.  But nevertheless: in this case I compare the installation with Conda like renting a van to move a bottle of water...  It installs thousands of files and makes things complex - only few files are really needed...  I hope to give you a smaller and more "clean" solution to install as "normal" user soon :-)  Cheers Norbert
